framework/src/constraints/MortarConstraintBase.C

```diff
@@ -81,6 +81,11 @@ MortarConstraintBase::validParams()
       "While DEFAULT quadrature order is typically sufficiently accurate, exact integration of "
       "QUAD mortar faces requires SECOND order quadrature for FIRST variables and FOURTH order "
       "quadrature for SECOND order variables.");
+  params.addParam<bool>(
+      "use_petrov_galerkin", false, "Whether to use the Petrov-Galerkin approach.");
```

Comment:
Since the actual use of PG depends on whether the constraint enforcement allows for using difference interpolation of the Lagrange multiplier variables, I'd expand on this description so that the user is more aware of it.